core/src/main/java/org/apache/calcite/rex/RexSimplify.java

```diff
@@ -2840,8 +2829,8 @@ static RexNode fix(RexBuilder rexBuilder, RexNode term) {
 
     @Override public String toString() {
       return "SEARCH(" + ref + ", " + (negate ? "NOT " : "") + rangeSet
-          + ", " + (nullTermCount + notNullTermCount)
-          + " terms of which " + nullTermCount + " allow null)";
+          + ", " + (nullAsTrueCount + nullAsFalseCount)
+          + " terms of which " + nullAsTrueCount + " allow null)";
```

Comment:
Is it helpful? Could it be replaced with "unknown as ..."?

Reply:
yes, I can improve this